app/(auth)/login/page.tsx

```diff
@@ -17,20 +16,21 @@ import { Button } from "@/components/ui/button";
 import { Input } from "@/components/ui/input";
 import { Label } from "@/components/ui/label";
 
+const authMethods = ["google", "email", "linkedin", "passkey"] as const;
+
+type AuthMethod = (typeof authMethods)[number];
+
```

Comment:
@mohdjami can you put these inside the `export default function Login()` function

Reply:
Done.